docs/index.html

```diff
@@ -220,8 +214,9 @@ <h4>CSS class i id</h4>
   .my-class{ background: aqua; }
   .my-tr-class{ color: deeppink; }
 </style>
-    <%= table_for User, @users, options: {id: "my-id", class: "my-class", tr_class: "my-tr-class" } %>
+    <!-- <%= table_for User, @users, options: {id: "my-id", class: "my-class", tr_class: "my-tr-class" } %> -->
```

Comment:
There is any explanation to "comment" this line? I mean, this is an HTML file, it will not process ERB code. Correct me if I'm wrong.